src/controllers/userProfileController.js

```diff
@@ -685,7 +685,17 @@ const userProfileController = function (UserProfile) {
             'update',
           );
         })
-        .catch((error) => res.status(400).send(error));
+        .catch((error) => {
+          if (error.name === 'ValidationError' && error.errors.lastName) {
+            const errors = Object.values(error.errors).map(er => er.message);
+            return res.status(400).json({
+              message: 'Validation Error',
+              error: errors,
+            });
+          }
+            console.error('Failed to save record:', error);
```

Comment:
Hey @DiyaWadhwani,
I think this console statement wouldn't be needed right? If I have got it wrong sorry for the trouble.